src/common/i18n/translations/en.ts

```diff
@@ -86,10 +86,10 @@ export const en: Translations = {
     previouslyRedeemed: "Previously redeemed",
     showMore: "Show more",
     showLess: "Show less",
-    today: "for today",
-    limitReachedRecent: "Limit reached %{time} ago %{today}.",
-    limitReachedDate: "Limit reached on %{dateTime} %{today}.",
-    limitReached: "Limit reached %{today}.",
+    today: " for today",
+    limitReachedRecent: "Limit reached %{time} ago%{today}.",
```

Comment:
I think there's supposed to be a space between `ago` and `%{today}`?

Reply:
added space

Reply:
@ilhaoni We moved the space over the `today` because `%{today}` is only shown in certain campaigns, and adding a space there would introduce a space in campaigns where `%{today}` is not shown.

@shermainelim Please make sure that all the `limitReached` translation copies do not have a space before `%{today}` and that `today` has a space before the copy like so: `" for today"`.

Reply:
ah ok, thanks @yongjiajie for explaining!

@shermainelim sorry I want to check in, did this slipped your mind, or you're not really sure about the reason of why we removed the space? This is currently just a small display change, so it's no biggie, but I do wanna highlight that in software engineering, reviewers will ask questions about the PR, and we do expect the writers of the PR to be able to explain the changes that was made.

Reply:
if it's a valid change, then no changes will be required. then both reviewers and writer will learn also.

Reply:
made a mistake here. Noted.